lib/src/main/java/com/orange/ods/compose/component/button/OdsIconToggleButton.kt

```diff
@@ -44,8 +40,9 @@ import com.orange.ods.theme.colors.OdsColors
  *
  * @param checked whether this IconToggleButton is currently checked
```

Comment:
`IconToggleButton` should be replaced with `OdsIconToggleButton`.

Same comment for documentation of `OdsIconToggleButtonsRow`.